packages/react/src/components/header/Header.stories.tsx

```diff
@@ -337,56 +342,181 @@ export const MinimalWithLocalization = (args) => {
         </Header.ActionBarItem>
       </Header.ActionBar>
 
       <Header.NavigationMenu>
-        <Header.Link href="#" label="Sosiaali- ja terveyspalvelut" />
-        <Header.Link href="#" label="Kasvatus ja koulutus" />
-        <Header.Link href="#" label="Asuminen" />
+        <Header.Link
+          href="#"
+          label="Sosiaali- ja terveyspalvelut"
+          onClick={(event) => event.preventDefault()}
+          active
+          dropdownLinks={[
+            <Header.Link
+              href="#"
+              label="Terveydenhoito"
+              active
+              dropdownLinks={[
+                <Header.Link href="#" label="Hammashoito" />,
+                <Header.Link href="#" label="Julkinen terveydenhoito" />,
+              ]}
+            />,
+            <Header.Link
+              href="#"
+              label="Senioripalvelut"
+              dropdownLinks={[
+                <Header.Link href="#" label="Viriketoiminta" />,
+                <Header.Link href="#" label="Kuntouttavat palvelut" />,
+              ]}
+            />,
+          ]}
+        />
+        <Header.Link
+          href="#"
+          label="Kasvatus ja koulutus"
+          dropdownLinks={[
+            <Header.Link
+              href="#"
+              label="Kasvatus"
+              active
+              dropdownLinks={[
+                <Header.Link href="#" label="Varhaiskasvatus" />,
+                <Header.Link href="#" label="Esiopetus" />,
+              ]}
+            />,
+            <Header.Link
+              href="#"
+              label="Koulutus"
+              dropdownLinks={[
+                <Header.Link href="#" label="Perusopetus" />,
+                <Header.Link href="#" label="Toisen asteen koulutus" />,
+                <Header.Link href="#" label="Työväenopistot" />,
+              ]}
+            />,
+          ]}
+        />
+        <Header.Link
+          href="#"
+          label="Yritykset ja työ"
+          dropdownLinks={[
+            <Header.Link
+              href="#"
+              label="Työnantajat"
+              active
+              dropdownLinks={[<Header.Link href="#" label="Yritykset" />, <Header.Link href="#" label="Yrittäjät" />]}
+            />,
+            <Header.Link
+              href="#"
+              label="Työntekijät"
+              dropdownLinks={[<Header.Link href="#" label="Avoimet työpaikat" />]}
+            />,
+          ]}
+        />
       </Header.NavigationMenu>
     </Header>
-  );
+    <div id="content" />
+  </>
+);
+
+WithFullFeaturesCustomTheme.args = {
+  theme: {
+    '--color-focus-outline': 'var(--color-metro-dark)',
+    '--header-color': 'var(--color-black)',
+    '--actionbar-background-color': 'var(--color-engel)',
+    '--nav-background-color': 'var(--color-engel-light)',
+    '--nav-border-color': 'var(--color-black)',
+    '--nav-link-hover-color': 'var(--color-black)',
+    '--universal-bar-background-color': 'var(--color-black-20)',
```

Comment:
> If we want to make the example prettier and -ofc- accessible let's make a tix for designers. Maybe @laurakarhu can weigh in.

I am rooting for making this story and doc site example of custom theme more prettier because this universal background colour in particular is killing me 😆 Could it be just `var(--color-white)` here? @nooranen

Reply:
😁 just tell me what 😆

Reply:
I'll say it's good enough now, I'll create tickets for designers to create prettier customisation examples later on.